.docker.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+set -x
+
+env
+storage-ng-travis-cpp && bash <(curl -s https://codecov.io/bash)
```

Comment:
yeah, I even plan to add there condition to early quit if "TRAVIS" env variable is not set. But it is still POC and yet it do not work, so it is just testing service for computing coverage for c++ project.